torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -1273,7 +1292,13 @@ def __init__(self, device, use_less_ttnn_op_types):
 
     def call(self, gm: torch.fx.GraphModule):
         # Decompose some aten ops to simpler aten ops
-        gm = rewrite_graph(gm, decompose_aten_to_aten_ops)
+        max_try = 10
+        cnt = 0
+        while cnt < max_try:
```

Comment:
I'm thinking if we should raise an exception when running out of tries. Because that means the graph still can't not converge and probably in a bad state

Reply:
ok I added